src/Maestro/tests/Maestro.ScenarioTests/ScenarioTests_Subscriptions.cs

```diff
@@ -0,0 +1,239 @@
+                    Target Repository URL: {repo2Uri}
+                    Target Branch: master
+                    Update Frequency: everyWeek
+                    Batchable: False
+                    Merge Policies:
+                    - Name: Standard
+                    ";
+
+                    await using AsyncDisposableValue<string> yamlSubscriptionId = await CreateSubscriptionAsync(yamlDefinition);
+
+                    Subscription expectedYamlSubscription = new Subscription(Guid.Parse(yamlSubscriptionId.Value), true, repo1Uri, repo2Uri, targetBranch);
+                    expectedYamlSubscription.Channel = new Channel(42, channel1Name, "test");
+                    expectedYamlSubscription.Policy = new SubscriptionPolicy(false, UpdateFrequency.EveryWeek);
+                    MergePolicy standardPolicy = new MergePolicy { Name = "Standard" };
+                    expectedYamlSubscription.Policy.MergePolicies = ImmutableList.Create(standardPolicy);
+
+                    string expectedYamlSubscriptionInfo = SubscriptionStringBuilder.GetSubscriptionString(expectedYamlSubscription, "  - Merge Policies:\r\n    Standard");
+
+                    await ValidateSubscriptionInfo(yamlSubscriptionId.Value, expectedYamlSubscriptionInfo);
+                    await DeleteSubscriptionById(yamlSubscriptionId.Value);
+
+                    TestContext.WriteLine("Change casing of the various properties. Expecting no changes.");
+
+                    string yamlDefinition2 = $@"
+                    Channel: {channel1Name}
+                    Source Repository URL: {repo1Uri}
+                    Target Repository URL: {repo2Uri}
+                    Target Branch: master
+                    Update Frequency: everyweek
+                    Batchable: False
+                    Merge Policies:
+                    - Name: standard
+                    ";
+
+                    await using AsyncDisposableValue<string> yamlSubscription2Id = await CreateSubscriptionAsync(yamlDefinition2);
+
+                    Subscription expectedYamlSubscription2 = new Subscription(Guid.Parse(yamlSubscription2Id.Value), true, repo1Uri, repo2Uri, targetBranch);
+                    expectedYamlSubscription2.Channel = new Channel(42, channel1Name, "test");
+                    expectedYamlSubscription2.Policy = new SubscriptionPolicy(false, UpdateFrequency.EveryWeek);
+                    expectedYamlSubscription2.Policy.MergePolicies = ImmutableList.Create(standardPolicy);
+
+                    string expectedYamlSubscriptionInfo2 = SubscriptionStringBuilder.GetSubscriptionString(expectedYamlSubscription2, "  - Merge Policies:\r\n    standard");
+
+                    await ValidateSubscriptionInfo(yamlSubscription2Id.Value, expectedYamlSubscriptionInfo2);
+                    await DeleteSubscriptionById(yamlSubscription2Id.Value);
+
+                    TestContext.WriteLine("Attempt to add multiple of the same merge policy checks. Should fail.");
```

Comment:
This "test" appears to actually be a lot of tests mushed into one.   There is an "IgnoresCase" test, a "MultipleMergePolicyRejected" test... can we break these up so that the tests are a touch more granular than "something went wrong somewhere"?